docs/API.md

```diff
@@ -609,6 +608,43 @@ Returns a unix time in seconds, like `,"time":1493426328`.
 Returns an empty string. This function is used when the `timestamp` option
 is set to `false`.
 
+<a id=".destination"></a>
+# .destination(dest?)
+
+Create a pino destination. It yields a 30% over using a standard Node.js
```

Comment:
"It yields a stream with 30% more throughput than a standard Node.js stream."

or

"It returns a stream with significantly more throughput than a standard Node.js stream."